File: ExpenseSplitter.Frontend/ExpenseSplitter.Web/src/app/models/expense/expense-list.model.ts
import { ExpenseTypeEnum } from 'src/app/models/expense/expense-type.enum';

export class ExpenseListModel {
    id: number;
    name: string;
    type: ExpenseTypeEnum;
    paidAt: Date;
    payerName: string;
    isPaidByMe: boolean;
    value: number;
    iSpent: number;
}

File: ExpenseSplitter.Backend/ExpenseSplitter.Api/Models/Expenses/ExpenseListModel.cs

using System;

namespace ExpenseSplitter.Api.Models.Expenses
{
    public class ExpenseListModel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public ExpenseType Type { get; set; }
        public DateTime PaidAt { get; set; }
        public string PayerName { get; set; }
        public bool IsPaidByMe { get; set; }
        public decimal Value { get; set; }
        public decimal ISpent { get; set; }
    }
}
